cucumber/policy/features/support/policy_helpers.rb

```diff
@@ -32,68 +32,127 @@ def invoke status: nil, &block
   end
 
   def load_root_policy policy
-    conjur_api.load_policy("root", policy, method: Conjur::API::POLICY_METHOD_PUT)
+    resource('root').put(policy, header)
   end
 
   def update_root_policy policy
-    conjur_api.load_policy("root", policy, method: Conjur::API::POLICY_METHOD_PATCH)
+    resource('root').patch(policy, header)
   end
 
   def extend_root_policy policy
-    conjur_api.load_policy("root", policy, method: Conjur::API::POLICY_METHOD_POST)
+    resource('root').post(policy, header)
   end
 
-  def load_policy id, policy
-    conjur_api.load_policy(id, policy, method: Conjur::API::POLICY_METHOD_PUT)
+  def load_policy(id, policy)
+    resource(id).put(policy, header)
   end
 
-  def update_policy id, policy
-    conjur_api.load_policy(id, policy, method: Conjur::API::POLICY_METHOD_PATCH)
+  def update_policy(id, policy)
+    resource(id).patch(policy, header)
   end
 
-  def extend_policy id, policy
-    conjur_api.load_policy(id, policy, method: Conjur::API::POLICY_METHOD_POST)
+  def extend_policy(id, policy)
+    resource(id).post(policy, header)
   end
 
-  def make_full_id *tokens
-    super(tokens.join(":"))
+  def create_api_key role
+    login_resource.put(
+      "", Authorization: create_token_header, params: { role: role }
+    )
+  end
+
+  def admin_api_key
+    admin_resource.get
+  end
+
+  def get_login_token login, key
+    RestClient.post(
+      uri('authn', CGI.escape(login), 'authenticate'),
+      key, 'Accept-Encoding': 'Base64'
+    )
   end
 
-  def conjur_api
-    login_as_role('admin', admin_api_key) unless @conjur_api
-    @conjur_api
+  def admin_token
+    RestClient.post(
+      uri('authn', 'admin', 'authenticate'),
+      admin_api_key,
+      'Accept-Encoding': 'Base64'
+    )
+  end
+
+  def admin_resource
+    RestClient::Resource.new(
+     uri('authn', 'login', ''), 'admin', admin_password
+    )
+  end
+
+  def resource id
+    RestClient::Resource.new(
+      uri('policies', 'policy', id)
+    )
+  end
+
+  def login_resource
+    RestClient::Resource.new(
+      uri('authn', 'api_key', ''), 'admin', admin_password
+    )
+  end
+
+  def make_full_id *tokens
+    super(tokens.join(":"))
```

Comment:
I think this is confusing.  We have two methods of the same name, doing similar things, but with different signatures, and one over-riding the other and calling `super` on it.  

There is a larger point here that we want to avoid this kind of mixin programming too, but we don't need to address that yet since you'll be rewriting much of this.  But just keep that in the back of your mind.

Also, a point on naming: the `make_` here is superfluous (as it is in the existing method name in `FullId`).  To avoid that and to fix the confusion problem I mentioned above, I'd suggest:

`full_id` and `full_id_from_tokens`

The first change from `make_full_id` to `full_id` you don't need to make now if you don't want, but lets at least give the new one a new name.